ext/reflection/php_reflection.stub.php

```diff
@@ -204,6 +204,9 @@ public function getDeclaringClass(): ReflectionClass {}
     /** @tentative-return-type */
     public function getPrototype(): ReflectionMethod {}
 
+    /** @tentative-return-type */
```

Comment:
there's no need for the `@tentative-return-type` in case of new methods

Reply:
Will remove it, thanks @kocsismate. Just for clarification, because I did try and search this previously, what is the significance of this? I found some bits in an RFC, but it wasn't entirely clear.

Reply:
If you add the`@tentative-return-type` to a method, the return type would become tentative, as per https://wiki.php.net/rfc/internal_method_return_types. Tentative return types are useful for existing methods (for BC purposes), but unnecessary for new ones.

Reply:
Excellent, thanks for clearing that up.